.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+## Changes
+
+<!--
+  Required.
+
+  Briefly describe the changes introduced by this pull request.
+  Also link relevant issues with the "closes", "fixes" or "resolves" keywords,
+  and add co-authors where appropriate with the "co-authored-by" keyword.
+
+  Example:
+  Implemented the functionality to update the user's name.
+  Closes #42
+  Co-authored-by: @octocat
```

Comment:
This works outside of the commit description?

https://github.blog/2018-01-29-commit-together-with-co-authors/

Reply:
I guess that if we "squash and merge" the PR, then it ends up in the commit description, and that's our usual merging strategy, so it should work as intended.